imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -11,6 +11,7 @@
 
 import numpy as np
 import scipy.sparse as spr
+import tomli
```

Comment:
good point. I solved this now by 
1) removed the import tomli from dfm_metamod
2) created a class method called "from_file" in the ExchangeCollector class that openss the file and creates an instance of ExchangeCollector
3) in dfm_metamod it now calls ExchangeCollector.from_file instead of reading the toml file